stratified-sampling/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Stratified Sampling",
+  "description": "An implementation of stratified sampling",
+  "kind": "script",
+  "source_code": "script.whizzml",
+  "inputs": [
+    {
+      "name": "dataset",
+      "type": "dataset-id",
+      "description": "The dataset you wish to sample from"
+    },
+    {
+      "name": "field",
+      "type": "string",
+      "description": "The field id of the strata"
+    },
+    {
+      "name": "sizes",
+      "type": "map",
+      "description": "A map of field value to sample count"
```

Comment:
yet another nit: wouldn't "counts" be a better name? i'd even consider value-counts, but i'm fine if you don't prefer the shorter one.